tests/tracer/test_monkey.py

```diff
@@ -31,6 +31,18 @@ def test_patch_all_env_override_manual_patch(self):
         monkey.patch(sqlite3=True)
         assert "sqlite3" in monkey._PATCHED_MODULES
 
+    @run_in_subprocess()
+    def test_patch_raise_exception_manual_patch(self):
+        # Manual patching should not be affected by the environment variable override.
+        with self.assertRaises(monkey.ModuleNotFoundException) as me:
+            monkey.patch(module_dne=True)
+
+        assert (
+            """integration module ddtrace.contrib.module_dne does not exist, module will not have tracing available"""
```

Comment:
```suggestion
            "integration module ddtrace.contrib.module_dne does not exist, module will not have tracing available"
```

Reply:
what was wrong with the other module name?

Reply:
The string was 125 characters so I just shortened the module name so pass the flask8 checks

Reply:
ah, yeah, you could have broken it up on multiple lines (in fact `black` should have done that? I cannot remember if it does or not)

e.g.

```python
"this is a very long string that is too long"

# same as this

(
    "this is a very long "
    "string that is too long"
)
```


No reason to change back to the old name, just wanted to make sure it wasn't for another reason.